### DISCUSSION TITLE
How do I enable Slack Integration

### DISCUSSION THREAD

**Discussion by Alex-Giaquinto** [outline/outline#6640] [Self Hosting]
Hello,

I am looking to find out how to set up the Slack integration with my self hosted instances. I followed the documentation from here https://docs.getoutline.com/s/be25efd1-b3ef-4450-b8e5-c4a4fc11e02a which I found on some other threads. But I did what it said, added the two tokens the .env file requested and restarted the pm2 containers. But when I go back to the settings page, it still says it is disabled. Anyone got any ideas?

**Comment by tommoor** [comment]
Currently you need to enable Slack auth as well: https://docs.getoutline.com/s/hosting/doc/slack-sgMujR8J9J

**Comment by Alex-Giaquinto** [comment]
Great this fixed the issue. Appreciate the help! @tommoor ...Will it ever become available without the user authentication part? (Even though I guess that just becomes a webhook)

**Reply by tommoor** [reply]
It will, this needs separating definitely
